Tracking/src/TrackChecker.cc

```diff
@@ -430,8 +430,8 @@ void TrackChecker::getCollection(LCCollection* &collection, std::string collecti
     collection = evt->getCollection( collectionName ) ;
   }
   catch(DataNotAvailableException &e){
-    std::cout<<"- cannot get collections !!"<<std::endl;
-    streamlog_out(ERROR) << "Collection " << collectionName.c_str() << " is unavailable" << std::endl;
+    streamlog_out(WARNING) << "- cannot get collections !!" << std::endl;
+    streamlog_out(WARNING) << "Collection " << collectionName.c_str() << " is unavailable" << std::endl;
```

Comment:
Why do you want to only warn if collection is not there?

Reply:
Yeah, maybe better to keep the error, but I drop the first line, because it is redundant